### DISCUSSION TITLE
504 Gateway Time-out error - app publishing

### DISCUSSION THREAD

**Discussion by applicationuser** [Budibase/budibase#10370] [Help]
Hello ,

Receiving 504 Gateway Time-out time out error when publishing app , after multiple tries app publishes but pages accessing data return 504 Gateway Time-out errors. 

Budibase cloud , version: 2.5.6

Pages access data stored in Budibase internal table and has over ~13k records but only access/display subset of data at a time

Thanks,

**Comment by ConorWebb96** [comment]
Hey @applicationuser,

We have responded to your support email based around this. Going to close this discussion for now until we receive additional information which was requested via the email.

Thank you!
